cpp/test/Ice/proxy/AllTests.cpp

```diff
@@ -1221,6 +1221,43 @@ allTests(TestHelper* helper)
 
     cout << "ok" << endl;
 
+    cout << "testing proxy to string is not affected by locale settings... " << flush;
+    auto currentLocale = std::locale();
+    std::locale::global(std::locale("en_US.UTF-8"));
+
+    {
+        optional<ObjectPrx> p = communicator->stringToProxy("test -t -e 1.0:tcp -h localhost -p 10000 -t 20000");
+        pstr = communicator->proxyToString(p);
+        test(pstr == "test -t -e 1.0:tcp -h localhost -p 10000 -t 20000");
```

Comment:
What would a `en_US.UTF-8` generated look like?

Reply:
It would use a thousand separator.